dias/analyzers/dataset_analyzer.py

```diff
@@ -162,11 +172,24 @@ def run(self):
                 # if flag files are not available yet for stackfile, do not process the stackfile
                 with h5py.File(_file, "r") as f:
                     file_end = f["index_map/time"][-1]["ctime"]
-                    if flag_tend < file_end:
-                        self.logger.info(
-                            "Flags not available for {0}, yet. Skipping..".format(_file)
+
+                old = file_end < (
+                    datetime.datetime.utcnow() - datetime.timedelta(hours=24)
+                )
+                flag_not_available = flag_tend < file_end
```

Comment:
Is this variable used any more after the next line?

Reply:
I had it for readability, not for factorability! Because `flag_tend < file_end` is not very clear. I could use a comment instead.